### DISCUSSION TITLE
CDN with Transformation Presets queries

### DISCUSSION THREAD

**Discussion by idavydova-ais** [directus/directus#16721] [Feature Requests]
Hi everyone. 
I have a question: I need to use CDN for my static files, but I have the project settings like Transformation Presets. And my GET request looks like this: **_directus.example.com/assets/image1?key=banner_**
I did the Transformation Presets configuration to change the static file quality. 
But my CDN wants a GET request like this:  **_cdn/assets/image1_**, without any queries and I will get a static file with high quality.

Question:
What do i need to do that get a static file with Transformation Presets quality, but with CDN? Thanks

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
